packages/devtools_app/lib/src/screens/performance/tabbed_performance_view.dart

```diff
@@ -77,12 +77,24 @@ class _TabbedPerformanceViewState extends State<TabbedPerformanceView>
 
   @override
   Widget build(BuildContext context) {
+    final isOffline = offlineController.offlineMode.value;
+    final isFlutterApp = serviceManager.connectedApp!.isFlutterAppNow!;
+
+    var showFrameAnalysis = isFlutterApp;
+    var showRasterStats = isFlutterApp;
+    var showRebuildStats = FeatureFlags.widgetRebuildstats && isFlutterApp;
+    final offlineData = controller.offlinePerformanceData;
+    if (isOffline && offlineData != null) {
+      showFrameAnalysis = showFrameAnalysis && offlineData.frames.isNotEmpty;
+      showRasterStats = showRasterStats && offlineData.rasterStats != null;
+      showRebuildStats = showRebuildStats &&
+          offlineData.rebuildCountModel.locationStats.value.isNotEmpty;
```

Comment:
@jacob314 can you verify whether this is the proper "empty" state to check here?

Reply:
I would add a rebuildCountModel.isNotEmpty getter.
_rebuildsForFrame.isNotEmpty is probably the cleanest implementation.

Reply:
Is it possible that we have rebuilds that are not associated with a flutter frame?

Reply:
no. all rebuilds have an associated frame. It is plausible in the future that the aggregate counts could however be zero even though there are rebuilds for a frame as a user could clear the total aggregate counts (e.g. by manually clicking the clear/reset button in the UI).